### ISSUE TITLE
In Privacy settings, Hide mail Client still refers to K9 User Agent

### ISSUE THREAD

**Issue by wsmwk** [thunderbird/thunderbird-android#8529]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App

Thunderbird for Android

### App version

8.0

### Where did you get the app from?

None

### Android version

unknown

### Steps to reproduce

user report at https://support.mozilla.org/en-US/questions/1472738

### Expected behavior

"Thunderbird for Android"

### Actual behavior

"K9"

**Comment by shamim-emon** [comment]
@kewisch which is the missing expected behavior addressed here, when `Hide mail client` is checked ?
1. When  I send mail, receiver will not be able to see my mail client in his/her mail header?
2. I as as user won't see any mail client info in any of my received mails?

Asking this because 1 is working in my case (Thunderbird debug version : 10.0-SNAPSHOT) so is it no 2? or is no1 ? In-case it's no 1, please provide steps to re-produce, so I can re-produce and fix it.
Please let me know. Thank you!

**Comment by malteger** [comment]
I think no functionality is affected, but with en_GB localization it is still displaying K-9 in the UI for the user agent setting, see https://github.com/thunderbird/thunderbird-android/blob/8668fabd5cff6cbbe630de653e490be3219ad39f/legacy/ui/legacy/src/main/res/values-en-rGB/strings.xml#L599
It looks like there are also some other languages where this is still the case.

**Comment by TheMinefighter** [comment]
This has been changed in weblate 5 moths ago: https://hosted.weblate.org/translate/tb-android/app-strings/en/?checksum=e586b5d3bfa510df , but for some reason the strings.xml is not yet updated...

> I think no functionality is affected, but with en_GB localization it is still displaying K-9 in the UI for the user agent setting, see
> 
> [thunderbird-android/legacy/ui/legacy/src/main/res/values-en-rGB/strings.xml](https://github.com/thunderbird/thunderbird-android/blob/8668fabd5cff6cbbe630de653e490be3219ad39f/legacy/ui/legacy/src/main/res/values-en-rGB/strings.xml#L599)
> 
> Line 599 in [8668fab](/thunderbird/thunderbird-android/commit/8668fabd5cff6cbbe630de653e490be3219ad39f)
>  <string name="global_settings_privacy_hide_useragent_detail">Remove K-9 User-Agent from mail headers</string> 
> 
> It looks like there are also some other languages where this is still the case.

**Comment by cketti** [comment]
I marked translations for this string that included "K-9" as "needs editing" on Weblate just now.

> This has been changed in weblate 5 moths ago: https://hosted.weblate.org/translate/tb-android/app-strings/en/?checksum=e586b5d3bfa510df , but for some reason the strings.xml is not yet updated...

We use American English as our source language. The identifier for that is "en" on Weblate. "en_GB" is the British English translation that hasn't been updated on Weblate (until I removed the translation just now because it should be identical to the source string).
